package.json

```diff
@@ -171,7 +171,6 @@
         "./packages/labs/gen-wrapper-vue:test",
         "./packages/labs/nextjs:test",
         "./packages/labs/preact-signals:test",
-        "./packages/labs/react:test",
```

Comment:
This feels like a band-aid. The test files in packages/labs/react are now stale. I don't think we really have reason to keep all the things in the labs directory for graduated packages like the tests and rollup builds.

Reply:
Strongly agree.

Reply:
There are. I think they can be done as a separate chore task. Nothing super pressing.